src/main/java/com/hillel/items_exchange/dto/ChildDto.java

```diff
@@ -16,10 +17,11 @@
 @Builder
 @EqualsAndHashCode
 public class ChildDto {
+
     @PositiveOrZero(message = "{invalid.id}")
     private long id;
     @NotNull(message = "{invalid.not-null}")
-    private String sex;
+    private ChildGender sex;
```

Comment:
In my opinion it's better to name just `Gender`, because it can be used not only for children